src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -11,6 +22,8 @@ package org.centos.pipeline
 def setupStage(stage, sshKey) {
     echo "Currently in stage: ${stage} in setupStage"
 
+    // TODO: Either remove sshKey arg, or determine how to invoke second credentialsID and variable name based on arg.
+    // Currently having an sshKey isn't that useful as we're still hard-coding the public credentialsID entry
```

Comment:
Not sure this has to be changed but we can discuss after refactor

Reply:
ack